Drug_check_usage.txt

```diff
@@ -0,0 +1,2 @@
+Here's the usage for drug search:
```

Comment:
Hi @LeiChen0218 , do we need this file? The usage could perhaps go in the README?